assembl/static2/js/app/components/login/signupForm.jsx

```diff
@@ -94,10 +91,19 @@ class SignupForm extends React.Component<Props, State> {
   signupHandler = (e) => {
     e.preventDefault();
     const slug = getDiscussionSlug();
+    const { termsAndConditionsIsChecked, privacyPolicyIsChecked, userGuidelinesIsChecked } = this.state;
+    const acceptedCookies = {
+      ACCEPT_CGU: termsAndConditionsIsChecked,
+      ACCEPT_PRIVACY_POLICY_ON_DISCUSSION: privacyPolicyIsChecked,
+      ACCEPT_USER_GUIDELINES_ON_DISCUSSION: userGuidelinesIsChecked
+    };
+    const acceptedLegalContentsArray = Object.keys(acceptedCookies)
```

Comment:
Please be consistant, one time you call it acceptedLegalContentsArray, another acceptedLegalContents. You should avoid to include the type of the variable in the variable name.

Reply:
This is to differentiate it of its form as an object. The shape of the data in the backend ("ACCEPT_THIS", "REJECT_THAT") forces a lot of transformations and I try to name the variables as what shape they have to avoid confusion